lib/perl/Genome/Model/Tools/Analysis/DumpIgvXmlBasic.pm

```diff
@@ -0,0 +1,96 @@
+package Genome::Model::Tools::Analysis::DumpIgvXmlBasic;
+
+use strict;
+use warnings;
+
+use Genome;
+use Command;
+use IO::File;
+use Cwd qw(abs_path);
+use URI;
+use XML::Writer;
+
+class Genome::Model::Tools::Analysis::DumpIgvXmlBasic {
+    is => 'Command',
+    has => [
+        resource_files => {
+            type => 'HASH',
+            doc => 'A keyed list of files to display in the IGV session. Each key is a path to a file and the corresponding value is that label that should be displayed in the IGV session for that file.',
+        },
+        output_file => {
+            type => 'String',
+            is_optional => 0,
+            doc => 'Output XML file',
+        },
+        reference_name => {
+            type => 'String',
+            is_optional => 0,
+            doc => 'the name of the reference (in IGV) that the bams are aligned to. E.g. "b37" for build 37 or "reference"for our internal build36',
+        },
+    ]
+};
+
+sub help_brief {
+    "Makes an IGV session for review. Supports as many tracks as you want. Panels and track layout are not explicitly specified."
+}
+
+sub help_detail {
+    <<'HELP';
+This helps create IGV session files for manual reviewers
```

Comment:
Should the brief and the detail be switched? It's unusual for the "brief" version to be longer than the "detail" version.